datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVUtil.scala

```diff
@@ -37,20 +37,24 @@ import org.apache.carbondata.spark.util.CommonUtil
  */
 object MVUtil {
 
+  var counter = 0
```

Comment:
counter should not be at object level. It will change state during concurrent execution.

Reply:
okay. Passed counter through method. Please review